src/main/resources/application.yml

```diff
@@ -92,6 +92,6 @@ logging:
 
 cors:
   allowedOrigins:
-  - "*"
```

Comment:
Same as above.

In addition to that we may not need the `origins`-parameter in the `@CrossOrigin` annotation if this setting right here is used as its default value.